cli/viam/main.go

```diff
@@ -785,6 +787,9 @@ In order to use the shell command, the robot must have a valid shell type servic
 									},
 								},
 								Action: func(c *cli.Context) error {
+									// TODO: remove this warning message
```

Comment:
can we track the TODOs in jira somewhere?

Reply:
Yep! Filed RSDK-4377 and linked here.